locale/core.yml

```diff
@@ -220,40 +220,46 @@ core:
       upload_button: Choose an Image...
 
     users:
-      title: => core.ref.users
       description: A paginated list of all users on your forum.
 
-      total_users: "Total users: {count}"
-
       grid:
         invalid_column_content: Invalid
```

Comment:
The alphabetization requirement is recursive all the way down, so a bit more rearrangement is needed.

Reply:
That line totally flew over my head 🙈